### DISCUSSION TITLE
Release 10.7.7

### DISCUSSION THREAD

**Discussion by joshuaboniface** [jellyfin/jellyfin#6513] [General]
# Jellyfin 10.7.7

Stable hotfix release for 10.7.z release branch.

## New Features and Major Improvements

N/A

## Release Notes

N/A

## Changelog

GitHub Project: https://github.com/orgs/jellyfin/projects/27

### [jellyfin](https://github.com/jellyfin/jellyfin) [2]

 * #6512 [@thornbill] Preferences fix
 * #6274 [@thornbill] Restore max width and height params

### [jellyfin-web](https://github.com/jellyfin/jellyfin-web) [1]

 * jellyfin/jellyfin-web#2759 [@thornbill] Fix serviceworker paths

<hr /><em>This discussion was created from the release <a href='https://github.com/jellyfin/jellyfin/releases/tag/v10.7.7'>Release 10.7.7</a>.</em>

**Comment by willmodev** [comment]
Hola, en esta versión la 10.7.7, presento un error que al ingresar en la parte de **géneros** al bajar el scroll y subir nuevamente no me carga las imágenes de las caratulas de las películas.
![WhatsApp Image 2021-09-07 at 10 58 33 PM](https://user-images.githubusercontent.com/55415696/132444278-cdc8048c-45c6-4e6f-a3eb-07c619b4597f.jpeg)
.

**Reply by xarmadigi** [reply]
Es la versión del servidor o el cliente? Mira si en web te hace lo mismo

**Reply by willmodev** [reply]
Revise y tanto en el servidor como cliente pasa lo mismo :(
